lisp/mastodon-tl.el

```diff
@@ -293,11 +293,14 @@ also render the html"
                         (lambda (media-attachement)
                           (let ((preview-url
                                  (cdr (assoc 'preview_url media-attachement))))
-                            (mastodon-media--get-media-link-rendering
-                             preview-url)))
-                        media-attachements "")))
-    (if (not (equal media-string ""))
-        (concat "\n" media-string ) "")))
+                            (if mastodon-tl--display-media-p
+                                (mastodon-media--get-media-link-rendering
+                                 preview-url)
+                              (concat "Media::" preview-url ))))
+                        media-attachements "\n")))
```

Comment:
Why do we re-add the ugly newlines between images?  That's only needed if `mastodon-tl--display-media-p` is nil, or not?

Reply:
I still think we should not add newlines between images when we do show images...